api/src/main/java/com/bjs/bjsapi/controllers/StudentScoreController.java

```diff
@@ -92,4 +95,36 @@ private double calculateExtra(int distance) {
 		return extra;
 	}
 
-}
\ No newline at end of file
+    public int studentAge(Student student) {
+        Date actual = new Date();
+        int studentAge = (int) student.getBirthDay().getYear();
+        int actualYear = (int) actual.getYear();
+        studentAge = actualYear - studentAge + 1900;
+        return studentAge;
+    }
+
+    public String classification(Student student) {
+        int score = calculateScore(student);
+        int honor = 0;
+        int victory = 0;
+        if (studentAge(student) > 19) {
+            honor = classificationInformationService.getHonorValue(student.getFemale(), 19);
+            victory = classificationInformationService.getVictoryValue(student.getFemale(), 19);
+        } else {
+            honor = classificationInformationService.getHonorValue(student.getFemale(), studentAge(student));
+            victory = classificationInformationService.getVictoryValue(student.getFemale(), studentAge(student));
+        }
```

Comment:
The check if the student's age is greater than 19 could be moved to the methods of `ClassificationInformationService` (`getHonorValue` and `getVictoryValue`)

Reply:
Alright.